modules/gitops-release-notes-1-18-2.adoc

```diff
@@ -0,0 +1,82 @@
+== Errata updates
+
+[id="RHSA-2025:23203-gitops-1-18-2-security-update-advisory_{context}"]
+=== RHSA-2025:23203 - {gitops-title} 1.18.2 security update advisory
+
+Issued: 2025-12-15
+
+The list of security fixes included in this release is available in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2025:23203[RHSA-2025:23203]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-18-2_{context}"]
+== Fixed issues
+
+Progressive Sync status and UI stability improvements::
+With this update, multiple issues are resolved where applications generated by `ApplicationSets` incorrectly displayed the Progressive Sync status as `Unknown` in the UI. The Progressive Sync state is now displayed correctly, and when the feature is disabled, the UI hides the field entirely. This update also fixes UI crashes that occur when Progressive Sync was enabled but the `applicationStatus` field is not available in the `ApplicationSet` status.
++
+link:https://issues.redhat.com/browse/GITOPS-7798[GITOPS-7798]
+
+git-lfs dependency update::
+* With this update, the `git-lfs` dependency in the Argo CD container image is updated to `v3.7.1`.
++
+link:https://issues.redhat.com/browse/GITOPS-8142[GITOPS-8142]
+
+Redis version upgraded to v7.2.11::
+* With this update, the Redis version in the {gitops-title} is upgraded to `v7.2.11`.
++
+link:https://issues.redhat.com/browse/GITOPS-8019[GITOPS-8019]
+
+GitOps Operator pulls images from registry.redhat.io::
+Before this update, when the `.spec.version` field was set in the Argo CD CR, the {gitops-title} Operator incorrectly pulled upstream images from `quay.io`. With this update, this issue is fixed. The {gitops-title} Operator pulls images from `registry.redhat.io` and only uses certified images from the Red Hat registry.
++
+link:https://issues.redhat.com/browse/GITOPS-7789[GITOPS-7789]
+
+Revised rolling update strategy for HAProxy deployment::
+Before this update, the `openshift-gitops-redis-ha-haproxy` deployment failed to upgrade on a three-worker node cluster due to a combination of pod anti-affinity rules and the update strategy. With this update, the {gitops-title} Operator revises the rolling update strategy to fix this issue.
++
+link:https://issues.redhat.com/browse/GITOPS-8033[GITOPS-8033]
+
+Added ingress permissions for successful reconciliation::
+Before this update, the {gitops-title} Operator failed to reconcile when `.spec.applicationSet.webhookServer.route.enabled` was set to `true` in the Argo CD CR. This issue occurred because the {gitops-title} Operator lacked the required ingress permissions for `config.openshift.io`. With this update, the {gitops-title} Operator includes the necessary permissions and reconciles successfully.
```

Comment:
```suggestion
Before this update, the {gitops-title} Operator failed to reconcile when `.spec.applicationSet.webhookServer.route.enabled` was set to `true` in the Argo CD CR. This issue occurred because the {gitops-title} Operator lacked the required ingress permissions for `config.openshift.io`. With this release, the {gitops-title} Operator includes the necessary permissions and reconciles successfully.
```